Files/Program.cs

```diff
@@ -43,10 +41,14 @@ static void Main()
         public static async Task OpenShellCommandInExplorer(string shellCommand, int pid)
         {
             System.Diagnostics.Debug.WriteLine("Launching shell command in FullTrustProcess");
-            ApplicationData.Current.LocalSettings.Values["ShellCommand"] = shellCommand;
-            ApplicationData.Current.LocalSettings.Values["Arguments"] = "ShellCommand";
-            ApplicationData.Current.LocalSettings.Values["pid"] = pid;
-            await FullTrustProcessLauncher.LaunchFullTrustProcessForCurrentAppAsync();
+            if (App.Connection != null)
```

Comment:
App.Connection here is always null because it checks immediately at launch (hence the app will not work as shell replacement).

Reply:
Thanks, I reverted the OpenShellCommandInExplorer function to the original. There's no need to set up the AppService stuff when we're just going to kill the process

Reply:
The `return` is sufficient only for Debug mode. In release mode this does not close the window, hence why we need to kill the "zombie" process